libs/ktor2client-onbehalfof/build.gradle.kts

```diff
@@ -26,6 +27,7 @@ dependencies {
     api(project(":libs:etterlatte-token-model"))
 
     implementation(libs.navfelles.tokenclientcore)
+    implementation(project(":libs:saksbehandling-common"))
```

Comment:
Litt usikker på om jeg er helt fan av at denne nå skal ta inn `saksbehandling-common`. Føles litt feil at en rest-klient-helper skal gjøre det. Er det for å unngå å gjøre noe catching der vi gjør kallet?

Reply:
Ja eller det for å bli fanget opp av StatusPagesKonfigurasjon og logge feilen bedre. Enig i at det føles litt rart i dra inn hele common for en exception. Men spørsmålet er vel heller da om Exception klassen ligger på riktig sted?
Hva tenker ud @oyvindsh

Reply:
Vi kan gjerne flytte exception-klassen ut til et eget lib, den ble mer plassert der av convenience opprinnelig. Målet er egentlig å bruke våre egne exceptions (eller subclasse de som i dette tilfellet) der vi sender exceptions videre opp, siden vi da får en mer treffsikker logging (den prøver å unwrappe til første laget med cause)